modules/ROOT/pages/cookies.adoc

```diff
@@ -0,0 +1,64 @@
+= About Exchange Cookies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Anypoint Exchange uses cookies to save user settings, enhance user experience, and improve performance.
+
+In general, websites have required cookies necessary for the site to work correctly, functional cookies necessary for certain site functionality, functional cookies that are not necessary for any site functionality, and advertising cookies.
+
+Exchange requires some cookies to work correctly, including cookies for session details and authentication information.
+
+Exchange normally uses some functional cookies, including cookies for site analytics. Exchange does not require any functional cookies for site functionality.
+
+Content in Exchange documentation often uses advertising cookies. If a video embedded in an Exchange portal is hosted on a third-party site such as YouTube that uses advertising cookies, then the video is not visible to a website visitor who disables advertising cookies, and the visitor sees a message explaining this.
+
+== Cookie Consent Management
+
+Exchange provides public portal website visitors with cookie consent management and full documentation of all cookies, in compliance with marketing and privacy regulations.
+
+Some countries require websites not to use any non-essential cookie until after the website visitor has granted explicit consent.
+
+When a user in a country with this requirement visits an Exchange public portal for the first time, Exchange notifies the user about the cookies stored in the browser, and the user accepts the default settings or opens the cookie consent manager to configure their cookie settings. Exchange finds the user's country by IP address.
+
+The Exchange cookie preferences button enables users in all countries to open the cookie consent manager and review or modify their settings at any time.
+
+The cookie consent manager shows settings for required cookies, functional cookies, and advertising cookies, and general information about cookies.
+
+Website visitors use the cookie consent manager to view the cookies the site uses, view descriptions of all individual cookies and understand what these cookies are doing, and enable or disable functional and advertising cookies.
+
+== Vanity Domains
+
+If you enable a vanity domain for your Exchange public portal, enable cookie consent management with the xref:portal-vanity-domain.adoc[public portal vanity domain instructions].
+
+== Cookies Used by Exchange
```

Comment:
@mcampo I wanted to check this table with you.

In https://www.mulesoft.org/jira/browse/DOCS-9632 there was a link to https://docs.google.com/spreadsheets/d/1c9DDosW3kFdfR12vHdDwbb5I57lbuiw1NEeOw6VM3es/edit#gid=1373564656 and I tried to get the cookie info from there. I did see that there is another spreadsheet at https://docs.google.com/spreadsheets/d/1CBC0pVJoNBBaacQRQXM3c7DHIPvzOUVndUvkAfFnd1Q/ , which appears to be older.

Am I using the right spreadsheet? Is the table missing any info, and is there any incorrect info? What should I enter as the description for the cookies that currently have no description?

Thank you!

Reply:
@wwarner-salesforce https://docs.google.com/spreadsheets/d/1c9DDosW3kFdfR12vHdDwbb5I57lbuiw1NEeOw6VM3es/edit#gid=1373564656 is the one we should use 👍

Reply:
@hiuxmaycry could you try to get info about the cookies that don't have a description?

Reply:
Added description for those cookies 🙂

Reply:
That's perfect, and I believe the table is ready now. Thanks to you both! Please re-approve if this looks good to you. @hiuxmaycry @mcampo